rued-esp32/src/peripherals.rs

```diff
@@ -66,17 +70,35 @@ impl
     pub fn take() -> Self {
         let peripherals = Peripherals::take().unwrap();
 
+        let spi1 = SpiPeripherals {
+            spi: peripherals.spi2,
+            sclk: peripherals.pins.gpio18.into(), // SCK
+            sdo: peripherals.pins.gpio23.into(),  // COPI
+            // NOTE: Display does not need a valid SDI pin; therefore
+            // settings this to an unused GPIO.
+            sdi: peripherals.pins.gpio10.into(), // dummy (SDI is not needed)
+            cs: Some(peripherals.pins.gpio5.into()), // CS
+        };
+
+        let spi2 = SpiPeripherals {
+            spi: peripherals.spi3,
+            sclk: peripherals.pins.gpio17.into(),     // -
+            sdo: peripherals.pins.gpio26.into(),      // -
+            sdi: peripherals.pins.gpio19.into(),      // CIPO
+            cs: Some(peripherals.pins.gpio12.into()), // -
+        };
```

Comment:
These are valid GPIO but not wired up to any SD cards I can use.  This is just temporary.